mbuild/formats/lammpsdata.py

```diff
@@ -188,6 +290,7 @@ def write_lammpsdata(
             * epsilon_0
         )
         charges[np.isinf(charges)] = 0
+        # TODO: FIX CHARGE UNIT CONVERSION
```

Comment:
This PR does not include any changes to the charge unit conversions (so that comment should not be there). Upon deeper inspection, I just realized that none of the test cases in the LAMMPS writer (i.e. the both the existing tests and the ones that Zijie and I modified) ensure that charges are written out correctly in reduced units. All of the test cases focus on LJ parameters, bonds, angles, dihedrals, etc. 

I agree that this needs more attention.  @zijiewu3 What are your thoughts?